.claude/skills/ritual-scribe/SKILL.md

```
@@ -0,0 +1,618 @@
+
+- A **bare decorator**. `@step` takes no arguments. Not `@step()`, not
+  `@step(max_visits=3)`.
+- Takes **exactly one** parameter, annotated with a pydantic model — or a
+  **union of pydantic models**, when several steps route into it:
+
+  ```python
+  Red = LintFailure | SuiteFailure | BothRed
+
+  @step
+  async def repair(failure: Red) -> Transition: ...
+  ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)